synapse/models/risk.py

```diff
@@ -942,6 +944,16 @@ def getModelDefs(self):
                         'doc': 'The currency in which payment was demanded.'}),
 
                 )),
+                ('risk:technique:masquerade', {}, (
```

Comment:
Do we need a `reporter` property here?

Reply:
No, it would be appropriate to use a `<(seen)- meta:source` here but reporter properties are typically for more abstract concepts. :+1: